### PR TITLE
CLOUD-593 Lowered default resources for DS pods to allow for running …

### PR DESCRIPTION
**Pull request by dgoldssfo**
…a smaller clusters

### CODE REVIEW

File: helm/ds/values.yaml [hunk 1]
```diff
@@ -69,14 +69,18 @@ backup:
...
+
 # Resource limits.
-# These help for pod placement on a larger cluster to ensure the OpenDJ instance gets sufficient resources.
-# For production you will want to specify CPU limits to help Kube schedule the pods.
-resources:
- requests:
-  memory: 2000Mi
- limits:
-   memory: 2500Mi
+# These help for pod placement on a larger cluster to ensure the DS instance gets sufficient resources.
```
Comment (shankar-forgerock) [inline helm/ds/values.yaml]: s/on a larger cluster/in a larger cluster/

File: helm/ds/values.yaml [hunk 2]
```diff
@@ -69,14 +69,18 @@ backup:
...
- requests:
-  memory: 2000Mi
- limits:
-   memory: 2500Mi
+# These help for pod placement in a larger cluster to ensure the DS instance gets sufficient resources.
+# For production, you will want to specify CPU limits to help Kube schedule the pods.
+resources: {}
+
+# Sample values:
+#resources:
```
Comment (wstrange) [inline helm/ds/values.yaml]: How is this different?

File: helm/ds/values.yaml [hunk 3]
```diff
@@ -69,14 +69,18 @@ backup:
 #opendjJavaArgs: "-server -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:+UseCompressedOops -XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:MaxRAMFraction=2"
 
 opendjJavaArgs: "-Xmx1024m -XX:+UnlockDiagnosticVMOptions"
```
Comment (wstrange) [inline helm/ds/values.yaml]: Also remember to adjust this down as well

File: helm/ds/values.yaml (hunk 2 above)
Comment (dgoldssfo) [reply helm/ds/values.yaml]: In the original, we have:

resources:
 requests:
  memory: 2000Mi
 limits:
   memory: 2500Mi

Here I have

resources: {}

(which is what's in the opendj Helm chart)

I was assuming this defaulted to something low but maybe not; do I need to specify an explicit memory amount? (If so, what's the lowest we can go?)

Comment (wstrange) [reply helm/ds/values.yaml]: we ran into issues with this before during training.  It would be better to set lower limits (and also adjust java opts down)

**Pull request merged**